habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+
+# TODO: is there a better way to do this more generaly? Maybe pybind a cmake compiled DATA_DIR?
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_dir = os.path.abspath(os.path.join(dir_path, "../../data/")) + "/"
+
+# dict keyed by uids with download specs for various test assets and datasets
+data_sources = {
+    # "example-uid": {
+    #
+    # }
+    "habitat-test-scenes": {
```

Comment:
Instead of a nested dicts, would an attr.s class make more sense here?

Reply:
@aclegg3 Actually, if you want the class to be serializable to checkout Python dataclasses (there is a small backport we can add to requirements easily). This seems like a good case to use them here.